flash_examples/graph_classification.py

```diff
@@ -22,15 +22,17 @@
 from torch_geometric.datasets import TUDataset  # noqa: E402
 
 # 1. Create the DataModule
-dataset = TUDataset(root="data", name="KKI")
+dataset = TUDataset(root="data", name="KKI").shuffle()
```

Comment:
We shuffle before `val_split` anyway so shouldn't need it here :smiley:
```suggestion
dataset = TUDataset(root="data", name="KKI")
```